engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/AvroUtils.scala

```diff
@@ -45,6 +47,17 @@ object AvroUtils {
 
   private def parser = new Schema.Parser()
 
+  def parsePayloadToByteBuffer(payload: Array[Byte]): Validated[IllegalArgumentException, ByteBuffer] = {
```

Comment:
Will these methods be visible in GUI?

Reply:
Nope. In GUI we have access only to class methods. Object is kind of util stuff.

Reply:
You are right @lciolecki , but also this is very specific, confluent convention, that there is magic byte at the beginning. I know that we keep here also `keySubject` and `valueSubject` that probable is also quite specific, but maybe will be better to mave this one to `ConfluentUtils`?

Reply:
I will agree with you but.. ;) AvroUtils helper uses `AvroUtils.valueSubject` in `valueSchema` method.. The same case we have at keySchema method..

Reply:
It's ok for me - I will change it.